File: Server/Entyties/Models/Survey.cs
using System;

namespace Entities.Models
{
    public class Survey
    {
		public int Id { get; set; }
		
		public string Question { set; get; }
		
		public int IdCreator { get; set; }
		
		public bool SeveralAnswer { get; set; }

		public bool AddResponse { get; set; }

        public DateTime TimeCreate { get; set; }

		public DateTime AbilityVoteFrom { get; set; }

		public DateTime? AbilityVoteTo { get; set; }
	}
}

File: QandAProject/src/app/classes/survey.ts
import { Answer } from './answer';
import { UserForPublic } from './user-for-public';

export class Survey {

  public addResponse: boolean;

  public answers: Answer[];

  public id: number;

  public question: string;

  public user: UserForPublic;

  public timeCreate: Date;

  public abilityVoteFrom: Date;

  public abilityVoteTo: Date = null;

  public minCountVotes: number;

  public maxCountVotes: number = null;

  constructor(
    question: string,
    answers: Answer[],
    addResponse: boolean,
    abilityVoteFrom: Date,
    minCountVotes: number,
    maxCountVotes: number = null,
    abilityVoteTo: Date = null
    ) {
      this.abilityVoteFrom = abilityVoteFrom;
      this.minCountVotes = minCountVotes;
      this.maxCountVotes = maxCountVotes;
      this.abilityVoteTo = abilityVoteTo;
      this.addResponse = addResponse;
      this.question = question;
      this.answers = answers;
      this.id = 0;
  }
}
